public/app/features/alerting/unified/components/rule-viewer/tabs/Details.tsx

```diff
@@ -151,7 +158,18 @@ export const Details = ({ rule }: DetailsProps) => {
               <DetailText
                 id="last-evaluation-duration"
                 label={t('alerting.alert.last-evaluation-duration', 'Last evaluation duration')}
-                value={isPaused ? pausedIcon : `${evaluationDuration} ms`}
```

Comment:
why are we removing the paused icon?

Reply:
ok, because we are rendering [here](https://github.com/grafana/grafana/pull/102808/files#diff-75591e117446c574c9781209c3c0ca1c53a71909b5c997ac32e9b46dbc79a6bdL135) right?

Reply:
Yes sorry I didn't make it clear, this is in the second part of a ternary that is already checking for the paused state 😁